recipe/meta.yaml

```diff
@@ -1,36 +1,63 @@
-{% set cudnn_version = "7.6.5.32-1" %}
-{% set cudnn_version_x86 = "7.6.5.33-1" %}
 {% set cuda_version = cudatoolkit | replace('.*',"") %}
-{% set dev_sha256_ppc = "fdf3d23bfa3ba22bef9e508473ab20affc25d5b84614475d5cbf0b40573ab04b" %}
-{% set sha256_ppc = "378894e78783f6866e714d85afc199c69754da71cfe8312e0163b5397c713cf4" %}
-{% set dev_sha256_x86 = "644d9e4206371764893408bf961afceaa9ee63175c2670fe9f8577bf5397cf7a" %}
-{% set sha256_x86 = "317483416b6516c0fb3d38f103bdd315352625c2e16f75aa37bb2a9323dd34c1" %}
-
+{% set cudnnvars = {
+  "10.2": {
+    "os": "rhel7",
+    "checksums": {
+      "linux64": "24486c4069ad2b5b1fc8082db6bdbf6ad65825cac11a673a781a2289bad0aaf4",
+      "ppc64le": "c7456bc1d4ce982969a6b2aeae011275bcf8fbadf7dc7cbd136f5820f0ab5fe4",
+    },
+    "checksums-dev": {
+      "linux64": "5da88e8095c5ec7c911195ad930da8fc7dce213543e7360bdc073078190dfb6d",
```

Comment:
So, linux-64 sha256sum values are corresponding to cudnn8 on x86 for cuda 10.2??

Reply:
yes